Modal.css

```diff
@@ -0,0 +1,74 @@
+  .modal-container > a {
+    display: inline-block;
+    text-decoration: none;
+    background-color: var(--component-black-color);
+    color: var(--component-white-color);
+    padding: 1rem;
+    border-radius: 3px;
+    color: var(--component-container-color);
+  }
+  .modal-container {
+    align-items: center;
+    background-color: var(--component-container-color);
+    color: var(--component-black-color);
+    border-radius: 0.3rem;
+    display: flex;
+    gap: 3rem;
```

Comment:
due to the gap the justify center is a little left. 
![image](https://user-images.githubusercontent.com/89966036/153478056-367df5b4-e751-4ee3-ba4b-864d42cb947f.png)

Reply:
Done. Thanks for pointing it out.